core/src/main/java/com/digitalpebble/storm/crawler/filtering/URLFilters.java

```diff
@@ -96,43 +90,48 @@ public void configure(JsonNode jsonNode) {
         Iterator<JsonNode> filterIter = jsonNode.elements();
         while (filterIter.hasNext()) {
             JsonNode afilterNode = filterIter.next();
+            String filterName = "<unnamed>";
+            JsonNode nameNode = afilterNode.get("name");
+            if (nameNode != null) {
+                filterName = nameNode.textValue();
+            }
             JsonNode classNode = afilterNode.get("class");
-            if (classNode == null)
+            if (classNode == null) {
+                LOG.error("Filter {} doesn't specified a 'class' attribute",
+                        filterName);
                 continue;
+            }
             String className = classNode.textValue().trim();
+            filterName += '[' + className + ']';
+
             // check that it is available and implements the interface URLFilter
             try {
                 Class<?> filterClass = Class.forName(className);
                 boolean interfaceOK = URLFilter.class
                         .isAssignableFrom(filterClass);
                 if (!interfaceOK) {
-                    LOG.error("Class {} does not implement URLFilter",
-                            className);
+                    LOG.error("Filter {} does not implement URLFilter",
```

Comment:
Sure. The error message includes both the filter name and the class name. I thought it was easier to also include the filter's name rather than just the classname. But I can revert